Frameworks/UIKit/UILabel.mm

```diff
@@ -44,61 +44,70 @@ @implementation UILabel {
     UILineBreakMode _lineBreakMode;
     BOOL _adjustFontSize;
     float _minimumFontSize;
-    float _originalFontSize;
     int _numberOfLines;
     BOOL _isDisabled;
     BOOL _isHighlighted;
     UIBaselineAdjustment _baselineAdjustment;
     StrongId<WXCTextBlock> _textBlock;
 }
 
-- (void)adjustFontSizeToFit {
+- (void)_adjustFontSizeToFit {
```

Comment:
different frames, different starting/min font sizes, etc.

---
In reply to: [97400039](https://github.com/Microsoft/WinObjC/pull/1768#discussion_r97400039) [](ancestors = 97400039)